Sources/XcbeautifyLib/Renderers/OutputRendering.swift

```diff
@@ -565,4 +565,80 @@ extension OutputRendering {
     func formatTestingStarted(group: TestingStartedCaptureGroup) -> String {
         colored ? group.wholeMessage.s.Bold.f.Cyan : group.wholeMessage
     }
+
+    func formatSwiftTestingRunStarted(group: SwiftTestingRunStartedCaptureGroup) -> String {
+        colored ? group.message.s.Bold : group.message
+    }
+
+    func formatSwiftTestingRunCompletion(group: SwiftTestingRunCompletionCaptureGroup) -> String {
+        return "Test run with \(group.numberOfTests) tests passed after \(group.totalTime) seconds"
+    }
```

Comment:
Can you add an implementation for the GitHub Actions and TeamCity renderers? I think it'll help to annotate these as callouts for each.

Today, we use `info` (at least for GHA) to include the respective XCTest info in the run summary.

Reply:
I added for these, I didn't see for the other ones; Maybe I am missing something?